service.onidle/addon.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.onidle" version="0.0.4" name="OnIdle" provider-name="Lunatixz">
+    <requires>
+        <import addon="xbmc.python" version="2.26.0"/>
+    </requires>
+    <extension point="xbmc.python.script" library="settings.py"/>
+    <extension point="xbmc.service" library="service.py" start="startup"/>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Idle Sleep Timer</summary>
+        <description lang="en_GB">Trigger Kodi actions when idle while playing.</description>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 3, June 2007</license>
```

Comment:
Use a SPDX identifier for the license please: https://spdx.org/licenses/
I'll had this to the checker at some point